src/components/Profile/ProfileContainer.js

```diff
@@ -3,29 +3,32 @@ import { withRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
 import axios from 'axios';
 
-import { PROFILE_URL } from '../../utils/url-utils';
+import { MAIN_URL } from '../../utils/url-utils';
 import { DEFAULT_USER_ID } from '../../utils/constants';
 import { setUserProfile } from '../../redux/actions/actionCreators';
```

Comment:
You're importing the "setUserProfile" in the current score - it is now available everywhere in the current file.
However later in code you're desctructuring the same "setUserProfile" from props.
This shadows the previously declared "setUserProfile".
Therefore in such cases d not use destructuring for "setUserProfile", just do `this.props.setUserProfile`;

This applies to all other files in your project.
Also this will fix the "...is already declared in the upper scope" linting error.

Reply:
/fxd